src/dmd/stmtstate.d

```diff
@@ -1,17 +1,16 @@
 /**
  * Used to help transform statement AST into flow graph.
  * Compiler implementation of the
```

Comment:
?

Reply:
Need to know where the 'break' statement winds up, for example.